upmc-enterprises/cda2fhir
# Feature/upmc 223 dedup organization

#### What does this PR do?

- Adds organization deduplication
- Also updated medication deduplication to differentiate between the duplicate medications that have different manufacturers or none. 
  e.g. 
   - medication without manufacturer org !== medication with manufacturer org and
   - Two of the same medications with different manufacturers are not equal. 

#### Related JIRA tickets:

UPMC-223 && UPMC-330

#### How should this be manually tested?

Check that 
a) medications that go through are deduplicated when they have the same medication code and the same organization (or both do not have an organization)
b) check that medications are not deduplicated when these conditions are not met
c) check that organizations are deduplicated from every location they occur. 

#### Background/Context:

#### New JIRA tickets for clinical review required?
